lizmap/www/assets/js/map.js

```diff
@@ -3527,6 +3527,18 @@ var lizMap = function() {
     var unitsRatio = OpenLayers.INCHES_PER_UNIT[units];
     var w = size.width / 72 / unitsRatio * aScale / 2;
     var h = size.height / 72 / unitsRatio * aScale / 2;
+    var printInProjectProjection = true;
+    if ('printInProjectProjection' in lizMap.config.options && lizMap.config.options.printInProjectProjection != 'True') {
+      printInProjectProjection = false;
+    }
+    if (printInProjectProjection
+      && lizMap.config.options.qgisProjectProjection.ref != lizMap.config.options.projection.ref
+    ) {
+      var printDpi = parseFloat(document.querySelector('#print-dpi').value);
```

Comment:
`printDpi` is never used.

Reply:
Could not test if we needed to use the hardcoded DPI 96 or the value of the chosen DPI, since I encountered a bug with QGIS Server 3.16 which does not take the DPI parameter into account (exported PDF or images have the same size)